### PR TITLE
Run Gradle build on Windows and CircleCI

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

File: gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckerPlugin.java [hunk 1]
```diff
@@ -27,5 +27,5 @@
   public void apply(Project project) {
     project.getExtensions().create("linkageChecker", LinkageCheckerPluginExtension.class);
     project.getTasks().create("linkageCheck", LinkageCheckTask.class);
-  }
+//  }
```
Comment (suztomo) [inline gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckerPlugin.java]: Confirmed this syntax error the gradle-plugin module fails the builds

Screen Shot 2020-08-24 at 00 47 28

**Review by elharo — COMMENTED** [review]
windows is failing

**Comment by suztomo** [comment]
It's not handling the absolute path in Windows correctly.

```
com.google.cloud.tools.dependencies.gradle.ExclusionFileFunctionalTest > can suppress linkage errors listed in exclusion files (absolute path) FAILED
    org.spockframework.runtime.ConditionNotSatisfiedError at ExclusionFileFunctionalTest.groovy:137
```

**Comment by suztomo** [comment]
@elharo Now it passed the tests.

**Pull request merged**